serverlessworkflow/modules/ROOT/pages/getting-started/modules/create-your-first-workflow-service.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+:description: Creating first workflow service
+:keywords: creating, serverless, workflow
+:navtitle: Creating your first workflow service
+:page-aliases:
+
+[id="create-your-first-workflow-service_{context}"]
+= Creating your first workflow service
+
+As a developer, you can use {context} and create a `Hello World` application, which includes the following procedures:
+
+* xref:getting-started/modules/proc-boostrapping-the-project.adoc[Bootstrapping a project]
+* xref:getting-started/modules/proc-creating-workflow.adoc[Creating a workflow]
+* xref:getting-started/modules/proc-running-application.adoc[Running your workflow application]
+* xref:getting-started/modules/proc-testing-application.adoc[Testing your workflow application]
+
+Also, you can directly access an example application by cloning `serverless-workflow-examples/serverless-workflow-hello-world` from the link:{kogito_sw_examples_url}/serverless-workflow-hello-world[GitHub repository].
+
+This document describes how to create a workflow application that serves a `hello_world` endpoint. The workflow contains the following two states:
+
+* `Inject Hello World`: Injects a `Hello World` message into the response
+* `Inject Mantra`: Injects a `Mantra` message into the response
+
+.Example Hello World workflow
+image::../../getting-started/images/first-workflow/hello-world-workflow.png[]
+
+.Prerequisites
+* Java {java_min_version} is installed with `JAVA_HOME` configured appropriately.
+* Apache Maven {maven_min_version} is installed.
+* {quarkus_cli_url}[Quarkus CLI] or xref:tooling/kn-plugin-workflow-overview.adoc[Knative Workflow CLI] {kn_cli_version} is installed.
+* Visual Studio Code with https://marketplace.visualstudio.com/items?itemName=redhat.java[Red Hat Java Extension]
+and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-serverless-workflow-editor[Red Hat Serverless Workflow Editor] is installed to edit your workflows.
+
+For more information about the tooling and the required dependencies, see xref:getting-started/getting-familiar-with-our-tooling.adoc[Getting familiar with {context} tooling].
+
+ifeval::["{kogito_version_redhat}" != ""]
+include::../../../pages/_common-content/downstream-project-setup-instructions.adoc[]
+endif::[]
+
+include::proc-boostrapping-the-project.adoc[leveloffset=+1]
+include::proc-creating-workflow.adoc[leveloffset=+1]
+include::proc-building-application.adoc[leveloffset=+1]
+include::proc-running-application.adoc[leveloffset=+1]
+include::proc-testing-application.adoc[leveloffset=+1]
+
+
+.Additional resources
```

Comment:
```suggestion
== Additional resources
```
I think it was better when it was a new section. Now it looks like it is under the testing part.